posts/2025-05-04-ask/index.qmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "Ask and you shall receive"
+description: |
+  Discussion of evolving complexity and value of the Bioconductor ecosystem. 
+author:
+  - name: Vince Carey
+    url: https://vjcitn.github.io/
+date: 2025-05-06
+creative_commons: CC BY
+categories:
+  - education
+image: "crazy.jpg"
+---
+title: "ask2025"
+author: "Vince Carey"
+---
+
+# In the beginning 
+
+... there was a 9-track tape shipped from
+Bell Labs.  S would be installed for "unix".  Extensions
+could be programmed in Fortran.  S REPORT presaged
+Sweave and knitr: troff, eqn, pic could be
+interlarded with S computations.  I have lost the associated
```

Comment:
I like the opener! Is "interlarded" a word?

Reply:
I am also not familiar with "interlarded" 🙂 . Wondering if you would like to add some more context here for those less familiar, for example:

> a 9-track tape shipped from Bell Labs to researchers contained the S programming language, designed for Unix systems. S was created to provide an interactive environment for statistical computing and data analysis, allowing researchers to explore data more flexibly and efficiently. Researchers could extend its capabilities using Fortran. Early tools like S REPORT foreshadowed modern literate programming tools such as Sweave and knitr, which integrate code and documentation to enhance reproducibility and clarity in data analysis.